Read_PIND/Read_PIND.ino

```diff
@@ -1,27 +1,31 @@
 /*
-* A proper header goes here
+* 
+* 
+* Hydrophones a,b,c, d will correspond to pins 3,4,5,6, respectively.
+*  Digital pin 7 corresponds to the duration indicator. 
 *
 */
 
 // Masks off digital pins 0, 1, and 2. 
-// Hydrophones a,b,c, d will correspond to pins 3,4,5,6, respectively.
-// Digital pin 7 corresponds to the duration indicator. 
 // Most significant bit of register corresponds to dignital pin 7
-uint8_t prevpinval = PIND & B11111000;
+uint8_t bitMask = B11111000;
+
+
+uint8_t prevpinval = PIND & bitMask;
 uint8_t pinval;
 uint8_t xorpins;
 extern volatile unsigned long timer0_overflow_count;
 unsigned long time_elapsed;
 
-// longs are 32 bits, so the array can only hold 218 values
+// longs are 32 bits, so the array can only hold 210 values
```

Comment:
Is this based on the amount of on-board memory that the board has? If so, perhaps we should reduce this a bit in order to allow for the SPI code to have some space to play around in down the road.

Reply:
Yes. It will have to reduced when SPI is added, not sure by how much.

Reply:
Works for me